ports/discord-game-sdk/vcpkg.json

```
@@ -3,6 +3,6 @@
   "version-string": "2.5.6",
   "port-version": 2,
```

```suggestion
  "version": "2.5.6",
  "port-version": 3,
```
This might be `version `here.

---

Addressed.